mobile/src/main/java/org/hawkular/client/android/backend/model/Operation.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.client.android.backend.model;
+
+import org.jboss.aerogear.android.core.RecordId;
+
+import com.google.gson.annotations.SerializedName;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+public class Operation implements Parcelable {
```

Comment:
Some operations have parameters defined. CHeck e.g. https://github.com/hawkular/hawkular-agent/blob/master/hawkular-wildfly-agent-feature-pack/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml#L619 (and the linked XSD). So in a future PR you may want to allow to edit the parameters before running the operation.